### DISCUSSION TITLE
Ignore articles (the, a) when sorting

### DISCUSSION THREAD

**Discussion by Bigmack3000** [advplyr/audiobookshelf#3347] [Ideas]
Is it possible to have abs ignore articles when sorting alphabetically? If not, can this be added? Thanks!

**Comment by nichwall** [comment]
This is available in the main server settings. If things are not sorted correctly with the setting enabled, specific examples as a bug report would be helpful in fixing it.

![Screenshot_20240830-224740.png](https://github.com/user-attachments/assets/f62bb530-3e4f-4726-af7a-59ca6ac7c643)

**Reply by Bigmack3000** [reply]
Thanks! Definitely skimmed right over that.  Very, very minor suggesting on it. I would love if the actually display name for the podcast remained the same.  Instead of seeing "Bright Sessions, The".  But again, minor issue.
